package-lock.json

```diff
@@ -4332,6 +4332,11 @@
           "requires": {
             "bn.js": "^4.11.1"
           }
+        },
```

Comment:
You have not installed any packages, so not needed to commit changes to package-lock

Reply:
Thanks